docs/commands/status.md

```diff
@@ -34,3 +34,40 @@ The following data is reported for each dependency when the YAML or JSON report
 - license: the dependency's SPDX license identifier
 - filename: the full path on disk to the dependency's cached metadata file, if available
 - errors: any error messages from failed status checks, if available
+
+## Status errors and resolutions
```

Comment:
> Adding some content to the status command documentation for the errors that can be returned for dependencies and common resolutions.

cc @mlinksva would you mind 👀  on this and let me know if the language here is appropriate and the suggestions are 👍 ?

Reply:
Made a couple small suggestions. Looks good to me overall!